libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -67,7 +68,7 @@ const AP_Param::GroupInfo AC_AutoTune_Heli::var_info[] = {
     // @Param: SEQ
     // @DisplayName: AutoTune Sequence Bitmask
     // @Description: 2-byte bitmask to select what tuning should be performed.  Max gain automatically performed if Rate D is selected. Values: 7:All,1:VFF Only,2:Rate D/Rate P Only(incl max gain),4:Angle P Only,8:Max Gain Only,3:VFF and Rate D/Rate P(incl max gain),5:VFF and Angle P,6:Rate D/Rate P(incl max gain) and angle P
-    // @Bitmask: 0:VFF,1:Rate D/Rate P(incl max gain),2:Angle P,3:Max Gain Only
+    // @Bitmask: 0:VFF,1:Rate D/Rate P(incl max gain),2:Angle P,3:Max Gain Only,4:Check Tune
```

Comment:
I wonder if we should consider always keeping the word order the same?  "Tune Check" vs "Check Tune"

Reply:
Fixed.  thanks.